workflow-controller/controller-service/src/main/java/io/hyscale/controller/manager/impl/LocalRegistryManagerImpl.java

```diff
@@ -132,5 +157,16 @@ private static String getHelperFunction(String registry) {
         String withHttpsAndSuffix = "https://" + registry + "/";
         return Arrays.asList(exactMatch, withHttps, withSuffix, withHttpsAndSuffix);
     }
-
+    
+
+    
+    private void validate(String path, boolean kubeConf, HyscaleErrorCode hyscaleErrorCode) throws HyscaleException {
+        File conffile = new File(path);
```

Comment:
1.camelcase confFile
2. No kubeconf here, remove it as a parameter
3. Cannot find docker config json as error code , no need of parameter